Kontent.Ai.Management/Modules/HttpClient/ManagementHttpClient.cs

```diff
@@ -59,10 +59,15 @@ internal class ManagementHttpClient : IManagementHttpClient
             }
 
             // Use the resilience logic.
-            var policyResult = await _resiliencePolicyProvider.Policy.ExecuteAndCaptureAsync(() =>
-                SendHttpMessage(messageCreator, endpointUrl, method, requestContent, headers));
-
-            response = policyResult.FinalHandledResult ?? policyResult.Result;
+            try
+            {
+                response = await _resiliencePolicyProvider.Policy.ExecuteAsync(() =>
+                    SendHttpMessage(messageCreator, endpointUrl, method, requestContent, headers));
+            }
+            catch(Exception)
+            {
+                throw;
```

Comment:
Is that the correct structure way to propagate the exception?  What is the difference between this and without the whole `try` and `catch` block?

Reply:
My bad, it is not necessary at all.

Reply:
fixed